src/lib/utils/SynapseClient.ts

```diff
@@ -887,9 +887,21 @@ export const getEntityHeaders = (
   references: ReferenceList,
   accessToken?: string,
 ) => {
+  // if references contains entity IDs with dot notation, fix the reference object
+  const fixedReferences = references.map(reference => {
```

Comment:
I saw another component getEntityHeaders failing because it was giving a list of entity IDs with dot notation, but didn't track down the component that was calling it.  I thought I'd just fix it here for good measure.